torch_geometric/nn/conv/point_transformer_conv.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, in_channels: Union[int, Tuple[int, int]],
                  out_channels: int, pos_nn: Optional[Callable] = None,
                  attn_nn: Optional[Callable] = None,
                  add_self_loops: bool = True, **kwargs):
-        kwargs.setdefault('aggr', 'mean')
+        kwargs.setdefault('aggr', 'add')
```

Comment:
Hmm this makes sense, but I guess its a breaking change for anyone already using it? What about making it an option?

Reply:
@Padarn I marked this as a breaking change in the changelog. I think we just definitely fix this since this change is for the better.